doc/user/zebra.rst

```diff
@@ -708,6 +708,25 @@ that sets the preferred source address, and applies the route-map to all
 
    ip protocol rip route-map RM1
 
+IPv6 example for OSPFv3.
+
+.. code-block:: frr
+
+   ipv6 prefix-list ANY seq 10 permit any
+   ipv6 prefix-list ANY seq 5 permit ::/0
```

Comment:
To avoid confusions for the reader can we leave an example as simple as it can be? I mean maybe ipv6 prefix-list with a single entry is enough?

Reply:
It doesn't work without both lines.  I forget why.     And at the time, I'd had enough of why it wasn't working.

Ok, so I've not tested it on latest FRR to see if some bug has been fixed.

Reply:
@ton31337 I did some tests and the permit ::/0 not required.